deploy/scripts/src/cri-orange/nino-check.ts

```diff
@@ -0,0 +1,138 @@
+import { iterationsStarted, iterationsCompleted } from '../common/utils/custom_metric/counter'
+import { group } from 'k6'
+import { SharedArray } from 'k6/data'
+import { type Options } from 'k6/options'
+import http, { type Response } from 'k6/http'
+import encoding from 'k6/encoding'
+import { selectProfile, type ProfileList, describeProfile } from '../common/utils/config/load-profiles'
+import { timeRequest } from '../common/utils/request/timing'
+import { isStatusCode200, isStatusCode302, pageContentCheck } from '../common/utils/checks/assertions'
+import { sleepBetween } from '../common/utils/sleep/sleepBetween'
+
+const profiles: ProfileList = {
+  smoke: {
+    ninoScenario1: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 1,
+      stages: [
+        { target: 1, duration: '5s' } // Ramps up to target load
+      ],
+      exec: 'ninoScenario1'
+    }
+  },
+  lowVolumeTest: {
+    ninoScenario1: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 900,
+      stages: [
+        { target: 30, duration: '5m' }, // Ramp up to 30 iterations per second in 5 minutes
+        { target: 30, duration: '15m' }, // Maintain steady state at 30 iterations per second for 15 minutes
+        { target: 0, duration: '5m' } // Total ramp down in 5 minutes
+      ],
+      exec: 'ninoScenario1'
+    }
+  }
+}
+
+const loadProfile = selectProfile(profiles)
+
+export const options: Options = {
+  scenarios: loadProfile.scenarios,
+  thresholds: {
+    http_req_duration: ['p(95)<=1000', 'p(99)<=2500'], // 95th percentile response time <=1000ms, 99th percentile response time <=2500ms
+    http_req_failed: ['rate<0.05'] // Error rate <5%
+  }
+}
+
+export function setup (): void {
+  describeProfile(loadProfile)
+}
+
+const env = { ipvCoreStub: __ENV.IDENTITY_CORE_STUB_URL }
+
+const stubCreds = {
+  userName: __ENV.IDENTITY_CORE_STUB_USERNAME,
+  password: __ENV.IDENTITY_CORE_STUB_PASSWORD
+}
+
+interface nino {
+  niNumber: string
+}
```

Comment:
```suggestion
interface nino {
  firstName: string
  lastName: string
  birthDay: string
  birthMonth: string
  birthYear: string
  niNumber: string
}
```
Update the interface type here to match the test data